nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d.hpp

```diff
@@ -114,6 +114,22 @@ class Costmap2D
     double win_size_x,
```

Comment:
We also need to add this to a migration guide and update the tutorials for the keepout / speed zones (and parameter guides)